### DISCUSSION TITLE
can't run the script - i get allways same error line 29 in manager.py

### DISCUSSION THREAD

**Discussion by entei1337** [swar/Swar-Chia-Plot-Manager#439] [Q&A]
(venv) C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main>python manager.py start
Traceback (most recent call last):
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\manager.py", line 29, in <module>
    start_manager()
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\commands.py", line 47, in start_manager
    raise ManagerError('Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.')
plotmanager.library.utilities.exceptions.ManagerError: Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.

-------------------------------------------

Traceback (most recent call last):
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\stateless-manager.py", line 58, in <module>
    jobs, running_work, next_job_work, next_log_check = monitor_jobs_to_start(
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\jobs.py", line 119, in monitor_jobs_to_start
    job, work = start_work(job=job, chia_location=chia_location, log_directory=log_directory)
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\jobs.py", line 169, in start_work
    process = start_process(args=plot_command, log_file=log_file)
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\processes.py", line 276, in start_process
    process = subprocess.Popen(
  File "C:\Users\PI\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\PI\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden #(system couldn#t find that file)

-----------------

How to solve this?

**Comment by entei1337** [comment]
i tryed to uninstall all python installation, and reinstalled phyton again, also i deleted the Swar-Chia-Plot-Manager-main folder. And extracted it again. I used same config.yaml as before. Still not works same Error

**Comment by dJani97** [comment]
Did you do Step 6) of the Installation guide?

> Copy config.yaml.default and name it as config.yaml in the same directory.

**Comment by entei1337** [comment]
yes i did and i started cmd and Powershell as admin:

PS C:\Windows\system32> cd C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main
PS C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main> pip install -r requirements.txt
Requirement already satisfied: dateparser==1.0.0 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 1)) (1.0.0)
Requirement already satisfied: discord_notify==1.0.0 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 2)) (1.0.0)
Requirement already satisfied: playsound==1.2.2 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 3)) (1.2.2)
Requirement already satisfied: psutil==5.8.0 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 4)) (5.8.0)
Requirement already satisfied: python_pushover==0.4 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 5)) (0.4)
Requirement already satisfied: PyYAML==5.4.1 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from -r requirements.txt (line 6)) (5.4.1)
Requirement already satisfied: python-dateutil in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from dateparser==1.0.0->-r requirements.txt (line 1)) (2.8.1)
Requirement already satisfied: tzlocal in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from dateparser==1.0.0->-r requirements.txt (line 1)) (2.1)
Requirement already satisfied: regex!=2019.02.19 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from dateparser==1.0.0->-r requirements.txt (line 1)) (2021.4.4)
Requirement already satisfied: pytz in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from dateparser==1.0.0->-r requirements.txt (line 1)) (2021.1)
Requirement already satisfied: requests>=1.0 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from python_pushover==0.4->-r requirements.txt (line 5)) (2.25.1)
Requirement already satisfied: chardet<5,>=3.0.2 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from requests>=1.0->python_pushover==0.4->-r requirements.txt (line 5)) (4.0.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from requests>=1.0->python_pushover==0.4->-r requirements.txt (line 5)) (1.26.4)
Requirement already satisfied: idna<3,>=2.5 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from requests>=1.0->python_pushover==0.4->-r requirements.txt (line 5)) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from requests>=1.0->python_pushover==0.4->-r requirements.txt (line 5)) (2020.12.5)
Requirement already satisfied: six>=1.5 in c:\users\PI\appdata\local\programs\python\python38\lib\site-packages (from python-dateutil->dateparser==1.0.0->-r requirements.txt (line 1)) (1.16.0)
PS C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main> python .\manager.py start
Traceback (most recent call last):
  File ".\manager.py", line 29, in <module>
    start_manager()
  File "C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main\plotmanager\library\utilities\commands.py", line 47, in start_manager
    raise ManagerError('Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.')
plotmanager.library.utilities.exceptions.ManagerError: Failed to start Manager. Please look at manager.log for more details on the error. It is in the same folder as manager.py.
PS C:\Users\PI\Desktop\Swar-Chia-Plot-Manager-main>

**Comment by entei1337** [comment]
Config should be correct it works already on a secound system (My old gaming Computer)
But this System is Fresh and new I just installed all drivers and the windows updates are up to date.

# This is a single variable that should contain the location of your chia executable file. This is the blockchain executable.
#
# WINDOWS EXAMPLE: C:\Users\Swar\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe
#   LINUX EXAMPLE: /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia
#  LINUX2 EXAMPLE: /home/swar/chia-blockchain/venv/bin/chia
chia_location: C:\Users\PI\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe


manager:
  # These are the config settings that will only be used by the plot manager.
  #
  # check_interval: The number of seconds to wait before checking to see if a new job should start.
  #      log_level: Keep this on ERROR to only record when there are errors. Change this to INFO in order to see more
  #                 detailed logging. Warning: INFO will write a lot of information.
  check_interval: 30
  log_level: ERROR


log:
  # folder_path: This is the folder where your log files for plots will be saved.
  folder_path: C:\Users\PI\Desktop\Chia_Log


view:
  # These are the settings that will be used by the view.
  #
  #            check_interval: The number of seconds to wait before updating the view.
  #           datetime_format: The datetime format that you want displayed in the view. See here
  #                            for formatting: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes
  # include_seconds_for_phase: This dictates whether seconds are included in the phase times.
  #        include_drive_info: This dictates whether the drive information will be showed.
  #               include_cpu: This dictates whether the CPU information will be showed.
  #               include_ram: This dictates whether the RAM information will be showed.
  #        include_plot_stats: This dictates whether the plot stats will be showed.
  check_interval: 30
  datetime_format: "%Y-%m-%d %H:%M:%S"
  include_seconds_for_phase: true
  include_drive_info: true
  include_cpu: true
  include_ram: true
  include_plot_stats: true


notifications:
  # These are different settings in order to notified when the plot manager starts and when a plot has been completed.

  # DISCORD
  notify_discord: false
  discord_webhook_url: https://discord.com/api/webhooks/0000000000000000/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

  # PLAY AUDIO SOUND
  notify_sound: false
  song: audio.mp3

  # PUSHOVER PUSH SERVICE
  notify_pushover: false
  pushover_user_key: xx
  pushover_api_key: xx
  
  # TWILIO
  notify_twilio: false
  twilio_account_sid: xxxxx
  twilio_auth_token: xxxxx
  twilio_from_phone: +1234657890
  twilio_to_phone: +1234657890


progress:
  # phase_line_end: These are the settings that will be used to dictate when a phase ends in the progress bar. It is
  #                 supposed to reflect the line at which the phase will end so the progress calculations can use that
  #                 information with the existing log file to calculate a progress percent.
  #   phase_weight: These are the weight to assign to each phase in the progress calculations. Typically, Phase 1 and 3
  #                 are the longest phases so they will hold more weight than the others.
  phase1_line_end: 801
  phase2_line_end: 834
  phase3_line_end: 2474
  phase4_line_end: 2620
  phase1_weight: 33.4
  phase2_weight: 20.43
  phase3_weight: 42.29
  phase4_weight: 3.88


global:
  # These are the settings that will be used globally by the plot manager.
  #
  # max_concurrent: The maximum number of plots that your system can run. The manager will not kick off more than this
  #                 number of plots total over time.
  max_concurrent: 10


jobs:
  # These are the settings that will be used by each job. Please note you can have multiple jobs and each job should be
  # in YAML format in order for it to be interpreted correctly. Almost all the values here will be passed into the
  # Chia executable file.
  #
  # Check for more details on the Chia CLI here: https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
  #
  # name: This is the name that you want to give to the job.
  # max_plots: This is the maximum number of jobs to make in one run of the manager. Any restarts to manager will reset
  #            this variable. It is only here to help with short term plotting.
  #
  # [OPTIONAL] farmer_public_key: Your farmer public key. If none is provided, it will not pass in this variable to the
  #                               chia executable which results in your default keys being used. This is only needed if
  #                               you have chia set up on a machine that does not have your credentials.
  # [OPTIONAL] pool_public_key: Your pool public key. Same information as the above.
  #
  # temporary_directory: Only a single directory should be passed into here. This is where the plotting will take place.
  # [OPTIONAL] temporary2_directory: Can be a single value or a list of values. This is an optional parameter to use in
  #                                  case you want to use the temporary2 directory functionality of Chia plotting.
  # destination_directory: Can be a single value or a list of values. This is the final directory where the plot will be
  #                        transferred once it is completed. If you provide a list, it will cycle through each drive
  #                        one by one.
  #
  # size: This refers to the k size of the plot. You would type in something like 32, 33, 34, 35... in here.
  # bitfield: This refers to whether you want to use bitfield or not in your plotting. Typically, you want to keep
  #           this as true.
  # threads: This is the number of threads that will be assigned to the plotter. Only phase 1 uses more than 1 thread.
  # buckets: The number of buckets to use. The default provided by Chia is 128.
  # memory_buffer: The amount of memory you want to allocate to the process.
  # max_concurrent: The maximum number of plots to have for this job at any given time.
  # max_concurrent_with_start_early: The maximum number of plots to have for this job at any given time including
  #                                  phases that started early.
  # stagger_minutes: The amount of minutes to wait before the next job can get kicked off. You can even set this to
  #                  zero if you want your plots to get kicked off immediately when the concurrent limits allow for it.
  # max_for_phase_1: The maximum number of plots on phase 1 for this job.
  # concurrency_start_early_phase: The phase in which you want to start a plot early. It is recommended to use 4 for
  #                                this field.
  # concurrency_start_early_phase_delay: The maximum number of minutes to wait before a new plot gets kicked off when
  #                                      the start early phase has been detected.
  # temporary2_destination_sync: This field will always submit the destination directory as the temporary2 directory.
  #                              These two directories will be in sync so that they will always be submitted as the
  #                              same value.
  - name: 1TB_ssd
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\temp
    temporary2_directory:
    destination_directory: Z:\
    size: 32
    bitfield: true
    threads: 24
    buckets: 128
    memory_buffer: 16000
    max_concurrent: 4
    max_concurrent_with_start_early: 7
    stagger_minutes: 30
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 60
    temporary2_destination_sync: false

  - name: 512GB_ssd
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: E:\temp
    temporary2_directory:
    destination_directory: Z:\
    size: 32
    bitfield: true
    threads: 24
    buckets: 128
    memory_buffer: 12000
    max_concurrent: 2
    max_concurrent_with_start_early: 7
    stagger_minutes: 60
    max_for_phase_1: 1
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
